core/src/lib.rs

```diff
@@ -469,11 +482,23 @@ pub enum AttributeBitFlags {
 }
 
 pub mod attribute {
-    #[derive(Clone,Debug,Eq,Hash,Ord,PartialOrd,PartialEq)]
+    use TypedValue;
+
+    #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialOrd, PartialEq)]
     pub enum Unique {
         Value,
         Identity,
     }
+
+    impl Unique {
+        // This is easier than rejigging DB_UNIQUE_VALUE to not be EDN.
```

Comment:
I don't get it: `pub const DB_UNIQUE_VALUE: Entid = 35;`.

Reply:
Remember that it technically isn't always `35` — it starts that way, certainly, and almost always is, but for external consumers and in the vocabulary API we want a `NamespacedKeyword`. `entids::DB_UNIQUE_VALUE = 35`, and in `values.rs` we have `lazy_static_namespaced_keyword_value!(DB_UNIQUE_VALUE, "db.unique", "value");` — which is an `edn::Value` — and this is the `NK` version.